ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -920,12 +920,26 @@ tests:
       ASSISTED_CONFIG: |
         KUBE_API=true
```

Comment:
I think we don't use that, `ENABLE_KUBE_API` is enough for setting the test in KubeAPI mode
```suggestion
```